.github/pull_request_template.md

```diff
@@ -0,0 +1,61 @@
+### Description
+
+<!--
+A description of the change in this PR.
+Place a reference to the Github issue or the PagerDuty incident, if any, here.
+-->
+
+### Type of Change
+
+- [X] Routine: low risk change
+- [ ] Declared: a significant or high risk change
+- [ ] Emergency: a change made to fix an on-going incident
+
+<!--
+Classify this change as one of the following, according to the change management
+policy in https://drive.google.com/drive/folders/1YgEceW5HQ2bQ4OAuhFIgN1oHmLk9RfTV
+-->
+
+### Testing
+
+- [ ] None: Implicit through tooling or non-configuration management change
```

Comment:
Technically, None shouldnt be an option for code other than documentation. So updating this to:

```suggestion
- [ ] None: Documentation change
```

Would be best